src/lib/fragments/entryProps.ts

```diff
@@ -41,5 +41,17 @@ export default gql`
         zip
       }
     }
+    ... on CallToAction {
+      callToActionDestination {
+        json
+        links {
+          assets {
+            hyperlink {
+              ...AssetProps
+            }
+          }
+        }
+      }
+    }
```

Comment:
As this has grown, I wonder if it's worth splitting this into separate fragments for each of the entry types (`ImageWrapper`, `Contact`, and `CallToAction`)? It seems like adding some unnecessary blot to queries and it'd be more explicit what we're expecting where if we broke it up